.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
   pull_request:
 jobs:
-  verify-terraform:
+  terraform-check:
```

Comment:
NIT: We should avoid bundling these changes with the lint commit

Reply:
Agreed, missed this on the rebase